src/routeGeneration/templateHelpers.ts

```diff
@@ -362,6 +366,130 @@ export class ValidationService {
     return new Buffer(value);
   }
 
+  public validateUnion(
+    name: string,
+    value: any,
+    fieldErrors: FieldErrors,
+    minimalSwaggerConfig: SwaggerConfigRelatedToRoutes,
+    subSchemas: TsoaRoute.PropertySchema[] | undefined,
+    parent = '',
+  ): any {
+    if (!subSchemas) { return; }
```

Comment:
I have the same question I have here as I did in comment https://github.com/lukeautry/tsoa/pull/400/files#r316279032

Reply:
https://github.com/lukeautry/tsoa/pull/400/commits/fb1f5231978893c73b3fcdd22118254f4bbf209c